### PR TITLE
chore: Remove link locale

### PR DESCRIPTION
**Pull request by nschonni**

### CODE REVIEW

File: windows/client-management/mdm/index.md [hunk 1]
```diff
@@ -46,7 +46,7 @@ For more details about the MDM policies defined in the MDM security baseline and
 
 - [MDM Security baseline for Windows 10, version 1809](http://download.microsoft.com/download/2/C/4/2C418EC7-31E0-4A74-8928-6DCD512F9A46/1809-MDM-SecurityBaseLine-Document-[Preview].zip)
```
Comment (illfated) [inline windows/client-management/mdm/index.md]:
```suggestion
 - [MDM Security baseline for Windows 10, version 1809](https://download.microsoft.com/download/2/C/4/2C418EC7-31E0-4A74-8928-6DCD512F9A46/1809-MDM-SecurityBaseLine-Document-[Preview].zip)
```

**Comment by illfated** [comment]
You may want to include the line
https://github.com/MicrosoftDocs/windows-itpro-docs/pull/4050/files#diff-74979e10892c5bc0099884b8ef9e6121R124
in your changes of `syntax -> xml` .
Unable to add suggested change there (out of bounds of the current changes).

File: windows/client-management/mdm/index.md (hunk 1 above)
Comment (nschonni) [reply windows/client-management/mdm/index.md]: Makes sense. The MS Docs Authoring pack cleanup doesn't handle the `download` subdomain right now https://github.com/microsoft/vscode-docs-authoring/blob/ad94714c64ad455903fc723dec04e1eb38c7dea9/docs-markdown/src/controllers/cleanup-controller.ts#L536

File: windows/client-management/mdm/index.md [hunk 2]
```diff
@@ -44,9 +44,9 @@ The MDM security baseline includes policies that cover the following areas:
 For more details about the MDM policies defined in the MDM security baseline and what Microsoft’s recommended baseline policy values are, see:
 - [MDM Security baseline for Windows 10, version 1903](http://download.microsoft.com/download/2/C/4/2C418EC7-31E0-4A74-8928-6DCD512F9A46/1903-MDM-SecurityBaseLine-Document.zip)
```
Comment (illfated) [inline windows/client-management/mdm/index.md]:
```suggestion
- [MDM Security baseline for Windows 10, version 1903](https://download.microsoft.com/download/2/C/4/2C418EC7-31E0-4A74-8928-6DCD512F9A46/1903-MDM-SecurityBaseLine-Document.zip)
```

**Comment by e0i** [comment]
@Justinha @lizap @mjcaparas @Dansimp 

Would you be able to review this PR that introduces mass changes to Markdown links to remove link locales and turn HTTP links to HTTPS? Thanks.

**Pull request merged**